tfs/src/main/java/hudson/plugins/tfs/listeners/JenkinsRunListener.java

```diff
@@ -54,4 +88,46 @@ private String getStartedBy(final Run run) {
         }
         return startedBy;
     }
+
+    private TeamGitStatus trySetPullRequestStatus(final Run run, @Nonnull final TaskListener listener, final GitStatusState buildState, final String buildDescription, final String targetUrl) {
```

Comment:
The return value of this method is never used, so you probably don't need to return anything.

Reply:
Changed it to void.